src/mudlet-lua/lua/LuaGlobal.lua

```diff
@@ -1,11 +1,24 @@
-----------------------------------------------------------------------------------
---- Mudlet Lua packages loader
-----------------------------------------------------------------------------------
+-- Mudlet Lua packages loader
```

Comment:
Changing to a single line comment is useful as then it shows up in the Lua error messages - instead of a long line of `-`...!